python/cuda_cccl/tests/parallel/examples/transform/binary_transform_object.py

```diff
@@ -34,7 +34,67 @@ def add_op(a, b):
     print("Binary transform object API example passed.")
 
 
+def plus_binary_transform_example():
+    """Demonstrate binary transform using well-known PLUS operation."""
```

Comment:
Should we modify the examples to replace the existing `add_op` examples where we define the operator in Python with PLUS? Ideally we want people to use well known operations instead of them defining their own, and keeping both `add_op` and `PLUS` will lead to confusion.

To show that we support custom operators we could add a different example that contains some logic, e.g., only add if the array element is even. This also applies to our other algorithms.

Reply:
Good suggestion. I've modified all examples to use well-known ops wherever possible, and also added a few examples that use even-add as suggested.